<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAEditor</class>
 <widget class="QMainWindow" name="QAEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Animation Editor</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="wCentral">
   <property name="windowTitle">
    <string>Qt Animation Editor</string>
   </property>
  </widget>
  <widget class="QDockWidget" name="dwFrames">
   <property name="windowTitle">
    <string>Frames</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dwFramesContents">
    <layout class="QVBoxLayout" name="hlFrames">
     <item>
      <widget class="QSplitter" name="sFrames">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QListView" name="lvFrames">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="selectionRectVisible">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QtTreePropertyBrowser" name="tpbFrames" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwHeader">
   <property name="windowTitle">
    <string>Animation Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dwHeaderContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QtTreePropertyBrowser" name="tpbHeader" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mbMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="mFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="aNew"/>
    <addaction name="aOpen"/>
    <addaction name="aSave"/>
   </widget>
   <widget class="QMenu" name="mEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="aUndo"/>
    <addaction name="aRedo"/>
   </widget>
   <addaction name="mFile"/>
   <addaction name="mEdit"/>
  </widget>
  <widget class="QDockWidget" name="dwSequences">
   <property name="windowTitle">
    <string>Sequence Frames</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="tvSequences">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="aNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="aOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="aSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="aUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="aRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header location="global">QtTreePropertyBrowser</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>reset()</slot>
  <slot>open()</slot>
  <slot>save()</slot>
 </slots>
</ui>
